### DISCUSSION TITLE
Is this project still alive?

### DISCUSSION THREAD

**Discussion by vsimoesbh** [serverless-nextjs/serverless-next.js#2593] [General]
Hello! Hope you all doing great.

I have several nextjs frontends running using this component but it worries me and my team that no updates were made recently, we made it work with nextjs 12 but we still stuck on serverless version 2. This component its a very solid way of deploying next on AWS and we trusted it from start. But now we are considering deploying to vercel so we can leverage nextjs 13 features and more importantly some security concerns are starting to rise as time goes on.

Knowing what the plans are so we can prepare ourselves would be a life saver.

As community and users we appreciate all the work and effort.

Thank you

**Comment by martg0** [comment]
Hi @vitonezz! I am in the same boat, but I have found an alternative called open-next. Currently, I am building a CDK script to deploy it to AWS for each environment. We are currently testing it to incorporate middleware and the new features of Next.js 13.

If you would like to take a look, here is the link to the CDK project with the instructions:

https://github.com/martg0/NextJS-Serverless-AWS-CDK

**Reply by vitorow** [reply]
Nice, youre keeping the functions on lambda@edge as well?

**Comment by IsaacTrevino** [comment]
This thread is helpful for anyone wondering where to go from here.

https://github.com/serverless-nextjs/serverless-next.js/issues/2497
